scripts/chopsticks/assethub-transfers/polkadot-polimec.yml

```diff
@@ -0,0 +1,92 @@
+db: ./db.sqlite
+
+endpoint: wss://rpc.polimec.org
+import-storage:
+  System:
+    Account:
+      # Politest Sudo
+      [
+        [
+          [
+            "0xba143e2096e073cb9cddc78e6f4969d8a02160d716a69e08214caf5339d88c42",
+          ],
+          { providers: 1, data: { free: "10000000000000000" } },
+        ],
+      ]
+
+  ForeignAssets:
+    Asset:
```

Comment:
Can you please add a comment remarking that this part of the `import-storage` config will be deleted as soon we configure the pallet on-chain?